src/Client.php

```diff
@@ -303,9 +303,9 @@ public function getAccountInfo(): array
      *
      * @return array
      */
-    public function revokeToken(): array
+    public function revokeToken()
     {
-        return $this->rpcEndpointRequest('auth/token/revoke', []);
+        return $this->voidEndpointRequest('auth/token/revoke');
```

Comment:
Because the revoke endpoint on api doesn't return anything, so rpcEndpointRequest fails since it return type is `array`.

That's the returned error.
> TypeError: Return value of rpcEndpointRequest() must be of the type array, null returned

Another alternative is removing the return typehint from rpcEndpointRequest, since PHP 7.0 doesn't support nullable return types ([php 7.1 does](http://php.net/manual/pt_BR/migration71.new-features.php#migration71.new-features.nullable-types))